src/content/docs/errors-inbox/browser-tab.mdx

```diff
@@ -0,0 +1,147 @@
+
+## Triage section [#triage-card]
+
+<img
+    title="errors inbox triage"
+    alt="A screenshot depicing the triage section of the browser group errors tab."
+    src={errorsinboxErrorsInboxTriageSection}
+/>
+
+The triage section links the specific error occurrence you're viewing to a [system-created error group](/docs/errors-inbox/errors-inbox/#groups) that has a unique fingerprint. Why does that matter? That unique fingerprint enables you to triage an error group using a [status](/docs/errors-inbox/errors-inbox/#status) update or [assignment](/docs/errors-inbox/errors-inbox/#assign). The system-created error groups are the ones you find on the **Triage** tab. For more info on how they are generated, see [How error groups work](/docs/errors-inbox/errors-inbox/#how-groups-work).
+
+## Stack trace [#stack-trace]
+
+<img
+    title="errors inbox stack trace"
+    alt="A screenshot depicting errors inbox stack trace"
+    src={errorsinboxErrorsInboxStackTrace}
+/>
+
+Stack traces can show you where in the code an error is originating from to help for quicker resolution. There are different ways to make sure that the code you’re viewing is unminified. You can manually upload source maps via the UI or via the API. 
+
+### Troubleshoot missing stack traces [#troubleshoot]
+
+ Some reasons why you may not see a stack trace for an error:
+    * If errors are not thrown, they will not produce a stack trace.
+    * The Javascript is hosted on a CDN or another external location, and New Relic can't see the errors originating from those scripts. This can be solved in some cases by using cross-origin resource sharing (CORS).
+    * It is an AngularJS error. For more on this, see [Missing Angular errors](/docs/browser/new-relic-browser/troubleshooting/angularjs-errors-do-not-appear).
+    * The error is a `SyntaxError` or `Script error`: These occur while the script is loading, so they do not produce a stack trace.
+    * Some older browsers may generate errors without a stack trace, or may provide a stack trace that is not usable.
+    * You've captured the error before the New Relic agent sees it.
+    * If the error occurs during multiple [harvest cycles](http://[cycle](docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/glossary#harvest-cycle)) in the same user visit, only error events reported during the first harvest cycle will have a stack trace recorded.
+
+### Solutions for reporting stack traces [#solutions]
+
+    * Use the [`noticeError` API call](/docs/browser/new-relic-browser/browser-agent-spa-api/notice-error).
+    * Enable [development mode](/docs/browser/new-relic-browser/troubleshooting/view-detailed-error-logs-browser).
+    * Use the **JS errors** UI page to get more information, including the [event log](#event-log). The **JS errors** page is designed to give you actionable information even without a stack trace.
```

Comment:
I'd add here the page: /docs/browser/new-relic-browser/browser-pro-features/javascript-errors-page-examining-errors-over-time